hexa/plugins/connector_iaso/models.py

```diff
@@ -0,0 +1,261 @@
+import typing
+from datetime import datetime
+from logging import getLogger
+
+from django.contrib.auth.models import AnonymousUser
+from django.db import models, transaction
+from django.db.models import Q
+from django.template.defaultfilters import pluralize
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from hexa.catalog.models import Datasource, Entry
+from hexa.catalog.sync import DatasourceSyncResult
+from hexa.core.models.base import BaseQuerySet
+from hexa.core.models.cryptography import EncryptedTextField
+from hexa.plugins.connector_iaso.api import get_forms_json
+from hexa.user_management import models as user_management_models
+from hexa.user_management.models import Permission
+
+logger = getLogger(__name__)
+
+
+class IASOAccountQuerySet(BaseQuerySet):
+    def filter_for_user(
+        self,
+        user: typing.Union[AnonymousUser, user_management_models.User],
+    ):
+        if not user.is_authenticated:
+            return self.none()
+        elif user.is_superuser:
+            queryset = self.all()
+        else:
+            queryset = self.filter(
+                iasopermission__team__in=user.team_set.all()
+            ).distinct()
+
+        return queryset
+
+
+class IASOAccount(Datasource):
```

Comment:
I'd say `class Account` - we usually don't prefix our models with the plugin name